app/swagger/requests/ppiu.rb

```diff
@@ -22,6 +22,37 @@ class PPIU
             end
           end
         end
+
+        operation :post do
+          extend Swagger::Responses::AuthenticationError
+
+          key :description, 'Update a veterans payment information'
+          key :operationId, 'postPaymentInformation'
+          key :tags, %w[form_526]
+
+          parameter :authorization
+
+          parameter do
+            key :name, :body
+            key :in, :body
+            key :description, 'Payment information to be updated for the user'
+            key :required, true
+
+            schema do
+              property :account_type, type: :string, example: 'Checking'
```

Comment:
Is there a _known_ array of potential types for `account_type`?  If yes, perhaps add an `enum` of the possible response types for the consumer?  Here is [an example](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/swagger/requests/backend_statuses.rb#L38) of that syntax.

Reply:
Yea, I had posed some questions to EVSS about this very thing (among others) and they just got back to me today about it :)